### PR TITLE
Add prettier to CI and format files

### PR DESCRIPTION
**Pull request by micheal-parks**
PNPM next up 🚀 :shipit:

### CODE REVIEW

File: web/frontend/package.json [hunk 1]
```diff
@@ -13,9 +13,9 @@
     "build-npm": "vite build --config vite.lib.config.ts && npm run build-types",
     "check": "svelte-check --tsconfig ./tsconfig.json",
     "preview": "vite preview",
-    "_prettier": "prettier --ignore-path=.eslintignore \"**/*.{ts,js,cjs,svelte,json,html,md}\"",
+    "_prettier": "prettier \"**/*.{ts,js,cjs,svelte,json,html,md}\"",
     "_eslint": "eslint \"**/*.{ts,js,svelte}\"",
-    "lint": "npm run _eslint",
+    "lint": "npm run _prettier --check && npm run _eslint",
     "format": "npm run _prettier --write && npm run _eslint --fix",
```
Comment (DTCurrie) [inline web/frontend/package.json:19]:
```suggestion
    "format": "npm run _prettier -- --write && npm run _eslint --fix",
```

I think we need the extra `--` to pass args through a `npm run` script.

Comment (micheal-parks) [reply web/frontend/package.json:19]: Thanks for catching!

**Review by DTCurrie — APPROVED** [review]
✅

**Comment by DTCurrie** [comment]
Approved, it's a lot to get through but I scanned it and nothing jumped out as concerning. This will make working in RC a lot easier. `Format on Save` is back on the menu!

**Review by ethanlook — CHANGES_REQUESTED** [review]
I checked:

1. `.prettierrc.cjs` matches App
2. `.prettierignore` matches App

Does format run in our workflows? I can't find where that is happening.

**Review by ethanlook — APPROVED** [review]

**Comment by micheal-parks** [comment]
Yep! `lint-web` runs in our workflows.

**Comment by ethanlook** [comment]
> Does format run in our workflows? I can't find where that is happening.

Confirmed `lint`: https://github.com/viamrobotics/rdk/blob/main/Makefile#L59-L68

which runs in `test_web_e2e`: https://github.com/viamrobotics/rdk/blob/a05a989d448bd206d0b530b3ef97991c0e559fe2/.github/workflows/test.yml#L213

**Pull request merged**